### PR TITLE
Add September 24 notes

### PR DESCRIPTION
**Pull request by julen**
Not much to report but if you feel there's something else worth commenting please say so.

### CODE REVIEW

File: 2016-09/september-14.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+* [Taras](https://github.com/ta2-1) (Translate House)
+
+
+### Ongoing work
+
+* Migrations: there are some integrity errors when upgrading from 2.5 and
+  switching from MyISAM to InnoDB
+  ([#5246](https://github.com/translate/pootle/issues/5246))
+* Work on due dates has started
+  ([feature/duedats](https://github.com/translate/pootle/tree/feature/duedates))
```
Comment (unho) [inline 2016-09/september-14.md:17]: No note about current work on vfolders?

Comment (julen) [reply 2016-09/september-14.md:17]: Feel free to propose something, I'm happy to add it.

Comment (unho) [reply 2016-09/september-14.md:17]: - Work on vfolders continues and will be landed soon so it unblocks stats and monolingual PRs.

Comment (julen) [reply 2016-09/september-14.md:17]: Added.

**Comment by unho** [comment]
lgtm

**Pull request merged**